src/Main/Main.css

```diff
@@ -0,0 +1,8 @@
+main {
+  display: flex;
+  flex-wrap: wrap;
+  margin-bottom: 40px;
+  background-size: cover;
+  height: calc(100vh - 80px);
+  align-content: flex-start;
```

Comment:
if you add `width: 100%` it may fix the slight background image issues on your page